internal/artifacts/module_arch_test.go

```diff
@@ -973,24 +956,33 @@ func getContentPath(contentType, source string) string {
 	return "not-existing-content"
 }
 
-func validateArchiveContents(expectedFilesInArchive []string, archiveLocation string, isExists bool) {
+func validateArchiveContents(expectedFilesInArchive []string, archiveLocation string) {
 	archiveReader, err := zip.OpenReader(archiveLocation)
 	Ω(err).Should(BeNil())
 	defer archiveReader.Close()
 	var filesInArchive []string
 	for _, file := range archiveReader.File {
-		if file.Mode().IsRegular() {
-			filesInArchive = append(filesInArchive, file.Name)
-		}
+		filesInArchive = append(filesInArchive, file.Name)
 	}
 	for _, expectedFile := range expectedFilesInArchive {
-		Ω(contains(expectedFile, filesInArchive)).Should(Equal(isExists), "Did not find "+expectedFile+" in archive")
+		Ω(contains(expectedFile, filesInArchive)).Should(BeTrue(), "Did not find "+expectedFile+" in archive")
 	}
 
-	if isExists {
-		for _, fileInArchive := range filesInArchive {
-			Ω(contains(fileInArchive, expectedFilesInArchive)).Should(Equal(isExists), "Did not find "+fileInArchive+" in expected")
-		}
+	for _, fileInArchive := range filesInArchive {
+		Ω(contains(fileInArchive, expectedFilesInArchive)).Should(BeTrue(), "Did not find "+fileInArchive+" in expected")
+	}
+}
+
+func validateArchiveContentsExcludes(unexpectedFilesInArchive []string, archiveLocation string) {
+	archiveReader, err := zip.OpenReader(archiveLocation)
+	Ω(err).Should(BeNil())
+	defer archiveReader.Close()
+	var filesInArchive []string
+	for _, file := range archiveReader.File {
+		filesInArchive = append(filesInArchive, file.Name)
+	}
+	for _, expectedFile := range unexpectedFilesInArchive {
+		Ω(contains(expectedFile, filesInArchive)).Should(BeFalse(), "Did not find "+expectedFile+" in archive")
```

Comment:
The error here is that the file was found although it shouldn't be in the archive

Reply:
fixed